content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -313,7 +320,7 @@ Then run the docker command to acquire and run the collector based on this
 configuration:
 
 ```shell
-docker run -p 4318:4318 \
+docker run -p 4317:4317 \
```

Comment:
intentional to change from http/protobuf to grpc?

Reply:
Yes, it simplifies the code and I didn't see a reason to use a non-default here.